daemons/controld/controld_schedulerd.c

```diff
@@ -238,6 +287,10 @@ controld_expect_sched_reply(xmlNode *msg)
 void
 controld_sched_cleanup()
 {
+    if (controld_sched_timer != NULL) {
+        mainloop_timer_del(controld_sched_timer);
+        controld_sched_timer = NULL;
+    }
```

Comment:
without detailed analysis, shouldn't it rather be:

1. `controld_expect_sched_reply(NULL);`
2. `mainloop_timer_del(controld_sched_timer);`

Would look more logical to me(?)

Reply:
Looking at it again, I realized that controld_expect_sched_reply(NULL) really should be done in pe_subsystem_free() before disconnecting the client. Pushed new commit.